definitions/private-law/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -128,5 +128,28 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "LiveFrom": "01/03/2023",
+    "CaseTypeID": "PRLAPPS",
+    "CaseEventID": "citizenCaseWithdraw",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "citizen",
+          "caseworker-privatelaw-systemupdate"
+        ],
+        "CRUD": "CRU"
+      },
+      {
+        "UserRoles": [
+          "caseworker-privatelaw-courtadmin",
+          "caseworker-privatelaw-judge",
+          "caseworker-privatelaw-la",
+          "caseworker-privatelaw-superuser"
```

Comment:
Double checking, do we need these roles to have Read access, as probably they wont be served until when we are allowing withdrawal.

"[SOLICITORA]",
"[SOLICITORB]",
          "[SOLICITORC]",
          "[SOLICITORD]",
          "[SOLICITORE]",
          "[SOLICITORF]",
          "[SOLICITORG]",
          "[SOLICITORH]",
          "[SOLICITORI]",
          "[SOLICITORJ]",

Reply:
Hi @AlokDatta, No it's not required for solicitors as the event is exclusively available for citizens.